### ISSUE TITLE
Method 47

### ISSUE THREAD

**Issue by SwistakPL** [hfiref0x/UACME#34]
I think method 47 doesn't work on Windows 10 17134.165. It just opens eventvwr.

**Comment by hfiref0x** [comment]
Do you have default full patch RS4 without any 3rd party software?

**Comment by SwistakPL** [comment]
Yeah, I think so

**Comment by hfiref0x** [comment]
Reproduced with 17134.191. It seems new fake COM entry ignored. I've tried it with MMC snap-in COM entry (used in uacme) and {B29D466A-857D-35BA-8712-A758861BFEA1} gpedit COM entry from original article, both were ignored. Well unsure which one cumulative update broken this, but it is really irrelevant. However this method is still working in the 18204 RS prerelease build from 21 July 2018. This case need more wait to see if the supposed fix will be ported to insider builds of future Windows 10 updates.

Thanks for your report, all future updates on this case will be posted here.
